File: EJERCICIO_4/EJERCICIOSJAVA/Student.java
package EJERCICIOSJAVA;


public class Student {
    protected String name;
    protected String lastName;
    protected String identification;
    protected String address;
    protected String telephone;
    protected String RH;

    public Student(String name, String lastName, String identification, String address, String telephone, String RH) {
        this.name = name;
        this.lastName = lastName;
        this.identification = identification;
        this.address = address;
        this.telephone = telephone;
        this.RH = RH;
    }

    public String getName() {
        return name;
    }

    public String getLastName() {
        return lastName;
    }

    public String getIdentification() {
        return identification;
    }

    public String getAddress() {
        return address;
    }

    public String getTelephone() {
        return telephone;
    }

    public String getRH() {
        return RH;
    }
}

File: EJERCICIO_4/EJERCICIOSPHP/Student.php
<?php
class Student{
    protected $name;
    protected $lastName;
    protected $identification;
    protected $address;
    protected $telephone;
    protected $RH;

    public function __construct($name, $lastName, $identification, $address, $telephone, $RH){
        $this->name = $name;
        $this->lastName = $lastName;
        $this->identification = $identification;
        $this->address = $address;
        $this->telephone = $telephone;
        $this->RH = $RH;
 }

 public function getName(){
    return $this->name;
}

public function getLastName(){
    return $this->lastName;
}

public function getIdentification(){
    return $this->identification;
}

public function getAddress(){
    return $this->address;
}

public function getTelephone(){
    return $this->telephone;
}

public function getRH(){
    return $this->RH;
}

}
?>